econml/ortho_forest.py

```diff
@@ -526,11 +586,13 @@ def parameter_estimator_func(Y, T, X,
             diagonal[:T_res.shape[1]] = 0
             reg = lambda_reg * np.diag(diagonal)
             # Ridge regression estimate
-            param_estimate = np.linalg.lstsq(np.matmul(weighted_XT_res.T, XT_res) + reg,
-                                             np.matmul(weighted_XT_res.T, Y_res.reshape(-1, 1)),
-                                             rcond=None)[0].flatten()
-            # Parameter returned by LinearRegression is (d_T, )
-            return param_estimate
+            linear_coef_estimate = np.linalg.lstsq(np.matmul(weighted_XT_res.T, XT_res) + reg,
+                                                   np.matmul(weighted_XT_res.T, Y_res.reshape(-1, 1)),
+                                                   rcond=None)[0].flatten()
+            X_aug = np.append([1], X_single)
```

Comment:
Is this legit if X_single is None?

Reply:
The correct way to handle this is to not set default values for `sample_weight` and `X_single` since those arguments must always be passed in the base class. I fixed the code to reflect that. I also added some explanations around what `X_single` means.